scripts/docker_run.sh

```diff
@@ -88,31 +88,30 @@ test_run_state() {
 test_run_updates() {
     CONTAINER_ID=$(docker ps -a -q --filter "ancestor=acceptance-test-${TAG_NAME}" | head -n 1)
 
+    RANDOM_SUFFIX=$(LC_ALL=C tr -dc 'a-z' </dev/urandom | head -c3)
+    NEW_IMG_NAME="${IMG_NAME}-${NEW_IMG_NAME}-${RANDOM_SUFFIX}"
```

Comment:
NIt: Can the Internal NEW_IMG_NAME be called PRDT_VERSION or some such.. 
NEW_IMG_NAME="${IMG_NAME}-${PRDT_VERSION}-${RANDOM_SUFFIX}"

since that seems to be what the var is really...  it might be clearer to understand what the values are.

Reply:
the IMG_NAME or the NEW, is just the name of your container, so it can be anything like
```
acceptance-test-{IMG_NAME} =  acceptance-test-LINUX or acceptance-test-k3s or acceptance-test-whateverfilteryouwant
```

thats why i left it like more generic!